CONTRIBUTING.md

```diff
@@ -75,10 +75,12 @@ The VM doesn't include:
    [vagrant-librarian-chef](https://github.com/jimmycuadra/vagrant-librarian-chef),
    [vagrant-omnibus](https://github.com/schisamo/vagrant-omnibus),
    [vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest):
+   [vagrant-newdisk](https://github.com/dbfit/vagrant-newdisk):
 
         vagrant plugin install vagrant-librarian-chef
         vagrant plugin install vagrant-omnibus
         vagrant plugin install vagrant-vbguest
+        vagrant plugin install vagrant-newdisk
```

Comment:
Ok. This makes things easy for now at least. Do some controller types have a maximum number of attached disks? E.g. could be end up with a base box in the future with multiple disks already attached and then this could fail?